app/src/main/java/org/p2p/wallet/moonpay/model/PaymentMethod.kt

```diff
@@ -0,0 +1,19 @@
+package org.p2p.wallet.moonpay.model
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+
+enum class Method {
```

Comment:
please put this inside PaymentMethod class or name it with some context
when searching or using auto-complete - Method is too ambigiuos name